southpark-search/app.py

```diff
@@ -81,7 +81,12 @@ def main(task, num_docs, top_k):
     workspace = os.environ["JINA_WORKSPACE"]
     if task == "index":
         if os.path.exists(workspace):
-            print(f"The directory {workspace} does already exist. Please remove it before indexing again.`")
+            import warnings
+            print(f'\n +---------------------------------------------------------------------------------+ \
+                    \n |                                   🤖🤖🤖                                        | \
+                    \n | The directory workspace already exists. Please remove it before indexing again. | \
```

Comment:
I like it, @cristianmtr once you do your example, we could add a reference to the new example in this warning

Reply:
Cool, Ill do it for the other examples too then